bin/make_test

```diff
@@ -13,6 +13,9 @@ tempdir=$(mktemp -d)
 slug="test-project"
 cookiecutter . --directory "$1" --output-dir "$tempdir" --no-input slug="$slug" "${@:2}"
 cd "$tempdir"/"$slug"
+if [[ ! -v CI ]]; then
+  make services
+fi
```

Comment:
The cookiecutter's tests now need to run `make services` in the newly generated project before running `make sure` (but don't run `make services` on CI)